trio/_core/_tests/test_multierror.py

```diff
@@ -283,7 +283,9 @@ def noop(_):
 
     # Simple pass-through of all exceptions
     m = make_tree()
-    with pytest.raises(MultiError) as excinfo:
+    with pytest.raises(  # noqa: SIM117  # multiple-with-statements
```

Comment:
I think combining the with statements (here and before) is probably fine, even if slightly less readable.

Reply:
Disabled combining with statements in 76d7413